template/README.md

```diff
@@ -1,137 +1,163 @@
+> :bulb: Below is a detailed description of the _whole_ startup procedure, both user changeable and not.
 
-#### Detailed startup procedure
-__Preloaded__ - Comes bundled with the box, not possible to change
+---
 
-__System provided__ - Provided by the system in automated processes, not possible to change
+#### Detailed Startup Procedure
+_box_ - Comes bundled with the box, not possible to change
 
-__User provided__ - Provided by the user to alter the box or template in some way
+_system_ - Provided by the system in automated processes, not possible to change
 
-1. `/home/vagrant/.env_default` - _preloaded_ - default variables
-1. `vagrant/.env` - _user provided_ - variables override [details](#pre-packaged-configuration-switches)
-1. `vagrant/.env_override` - _system provided_ - variables are overridden for test purposes
-1. `vagrant/dev/vagrant/conf/pre_ansible.sh` - _user provided_ - script running before ansible bootstrap procedure, [details](dev/vagrant/conf/pre_bootstrap/README.md)
-1. `vagrant/dev/vagrant/conf/pre_bootstrap/*.yml` - _user provided_ - pre bootstrap tasks, running before hashistack software starts, [details](dev/vagrant/conf/README.md)
-1. `/etc/ansible/bootstrap.yml` - _preloaded_ - verify ansible variables and software configuration, run hashistack software and MinIO, & verify that it started correctly
-1. `vagrant/conf/post_bootstrap/*.yml` - _user provided_ - poststart scripts, running after hashistack software has started, [details](dev/vagrant/conf/pre_bootstrap/README.md)
-1. `vagrant/dev/conf/pre_ansible.sh` - _user provided_ - script running after ansible bootstrap procedure, [details](dev/vagrant/conf/README.md)
-1. `vagrant/ansible/*.yml` - _user provided_ - ansible tasks included in playbook, [details](#pre-and-post-hashistack-procedure)
+_user_ - Provided by the user to alter the box or template in some way
 
-### Starting a box
-The vagrant box ships with a default startup scheme. `make` from this directory will start the box, and run all books in [dev/ansible](dev/ansible) in lexical order (NB: `playbook.yml` is run first, but is only used to run all other playbooks) after the bootstrap-process for the hashistack is done. In the [example](test_example/dev/ansible/playbook.yml) we use it to start terraform which then starts a nomad-job.
+| What | Provided by | Description |
+|:------------|:------------:|:-----|
+|1 `/home/vagrant/.env_default`|[ _box_ ]| default variables |
+|2 `vagrant/.env`|[ _user_ ]| variables override, see [Pre-packaged Configuration Switches](file:///home/m88614/projects/vagrant-hashistack/template/README.md#pre-packaged-configuration-switches) for details |
+|3 `vagrant/.env_override`|[ _system_ ]| variables are overridden for test purposes |
```

Comment:
Why some of the paths are absolute, some relative? (Just noticed it now. )

Reply:
Huh.. I thought I changed this. I _swear_ I changed it to `/vagrant`.

Might be that I copy-pasted the initial table-version. Anyways, I'll make them absolute again!

Reply:
Mine example was not correct :)

Reply:
Fixed!